datadog/resource_datadog_user.go

```diff
@@ -2,24 +2,13 @@ package datadog
 
 import (
 	"context"
```

Comment:
```suggestion
	"context"
	"log"
```

Reply:
The log import is used in the resource to print some helpful messages.

Reply:
sure, but let's keep it grouped with the stdlib imports :)